tests/sources/xml/test_dspace_dim.py

```diff
@@ -216,3 +244,231 @@ def test_dspace_dim_transform_with_optional_fields_missing_transforms_correctly(
+        == "Journal of Geophysical Research: Solid Earth 121 (2016): 5859-5879"
+    )
+
+
+def test_get_citation_transforms_correctly_if_fields_blank():
+    source_record = create_dspace_dim_source_record_stub(
+        '<dim:field mdschema="dc" element="identifier" qualifier="citation" />'
+    )
+    assert DspaceDim.get_citation(source_record) is None
+
+
+def test_get_citation_transforms_correctly_if_fields_missing():
+    source_record = create_dspace_dim_source_record_stub()
+    assert DspaceDim.get_citation(source_record) is None
+
+
+def test_get_content_type_success():
+    source_record = create_dspace_dim_source_record_stub(
+        """
+        <dim:field mdschema="dc" element="type">Moving Image</dim:field>
+        <dim:field mdschema="dc" element="type">Dataset</dim:field>
+        """
+    )
+    assert DspaceDim.get_content_type(source_record) == [
+        "Moving Image",
+        "Dataset",
+    ]
+
+
+def test_get_content_type_transforms_correctly_if_fields_blank():
+    source_record = create_dspace_dim_source_record_stub(
+        '<dim:field mdschema="dc" element="type" />'
+    )
+    assert DspaceDim.get_content_type(source_record) is None
+
+
+def test_get_content_type_transforms_correctly_if_fields_missing():
+    source_record = create_dspace_dim_source_record_stub()
+    assert DspaceDim.get_content_type(source_record) is None
+
+
+def test_get_contents_success():
+    source_record = create_dspace_dim_source_record_stub(
+        """
+        <dim:field mdschema="dc" element="description" qualifier="tableofcontents"
+        >Chapter 1</dim:field>
+        """
+    )
+    assert DspaceDim.get_contents(source_record) == ["Chapter 1"]
+
+
+def test_get_contents_transforms_correctly_if_fields_blank():
+    source_record = create_dspace_dim_source_record_stub(
+        '<dim:field mdschema="dc" element="description" qualifier="tableofcontents" />'
+    )
+    assert DspaceDim.get_contents(source_record) is None
+
+
+def test_get_contents_transforms_correctly_if_fields_missing():
+    source_record = create_dspace_dim_source_record_stub()
+    assert DspaceDim.get_contents(source_record) is None
+
+
+def test_get_contributors_success():
+    source_record = create_dspace_dim_source_record_stub(
+        """
+        <dim:field mdschema="dc" element="contributor"
+        qualifier="author">LaFountain, James R.</dim:field>
+        <dim:field mdschema="dc" element="contributor"
+        qualifier="author">Oldenbourg, Rudolf</dim:field>
+        <dim:field mdschema="dc" element="creator">Jamerson, James</dim:field>
+        """
+    )
+    assert DspaceDim.get_contributors(source_record) == [
+        timdex.Contributor(value="Jamerson, James", kind="Creator"),
+        timdex.Contributor(
+            value="LaFountain, James R.",
+            kind="author",
+        ),
+        timdex.Contributor(
+            value="Oldenbourg, Rudolf",
+            kind="author",
+        ),
+    ]
```

Comment:
Noting now, but don't think we should make these changes in this refactor, but this feels like a good opportunity at some point to normalized `kind` values.  Seeing `Creator` (title cased) and `author` (lowercase).  I know `author` comes from the XML itself, and it's always risky to programatically convert case.

Maybe normalization is some kind of post-processing / post record transform?  Again, not needed now, just noting for the future.

Reply:
I was almost certain we had a Jira ticket for this but we don't, I'll create an issue

Reply:
Thanks for creating the ticket.  Maybe when this field method refactor is done, before we launch into orchestration, we could meet and survey the tickets created?  Get a feel for anything we may want to also tackle during the orchestration?  or after?

Reply:
Before sounds like a good idea!